spec/security/audit_rules_spec.rb

```diff
@@ -0,0 +1,171 @@
+describe LinuxAdmin::Security::AuditRules do
+  def test_file_name
+    File.join(data_file_path("security"), "audit.rules")
+  end
+
+  def test_file_contents
+    File.read(test_file_name)
+  end
+
+  around(:each) do |example|
+    text = test_file_contents
+    example.run
+    File.open(test_file_name, "w") do |file|
+      file.puts(text)
+    end
```

Comment:
Also can be `File.write`
Also, you don't need :each...it's implied as the default.